cdn/dev/js/videoloader.js

```diff
@@ -0,0 +1,13 @@
+const thumbnailsList = document.querySelectorAll("#keymanThumbnails");
+
+window.addEventListener('DOMContentLoaded', () => {
+  thumbnailsList.forEach(each => {
+  	each.addEventListener('click', () => {
+
+  		if(each.hasAttribute('data-video')) {
+        let videoId = each.getAttribute('data-video')
+    		each.outerHTML = "<iframe width='600' height='315' src='" + videoId + "' allowfullscreen></iframe>";
+      }
+    })
+  })
+})
```

Comment:
Use an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) to prevent pollution of global namespace:
```suggestion
(function() {
  const thumbnailsList = document.querySelectorAll(".video-thumbnail");
  
  window.addEventListener('DOMContentLoaded', () => {
    thumbnailsList.forEach(thumbnail => {
  	  thumbnail.addEventListener('click', () => {
  
  		  if(thumbnail.hasAttribute('data-video')) {
          let videoId = thumbnail.getAttribute('data-video')
    		  thumbnail.outerHTML = "<iframe width='600' height='315' src='" + videoId + "' allowfullscreen></iframe>";
        }
      })
    })
  })
})();
```

* I've also renamed the variable `each` to `thumbnail` to better represent what it is.
* I renamed the selector from `#keymanThumbnails` to `.video-thumbnail` (a class rather than an id, and singular rather than plural, and 'video' instead of 'keyman' to clarify what it is for, and finally, using kebab-case instead of camelCase :grin:)